clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/EcsSpec.java

```diff
@@ -52,11 +61,40 @@
   protected static final String TEST_OPERATIONS_LOCATION =
       "src/integration/resources/testoperations";
   protected static final String TEST_ARTIFACTS_LOCATION = "src/integration/resources/testartifacts";
-  protected final String ECS_ACCOUNT_NAME = "ecs-account";
+  protected static final String ECS_ACCOUNT_NAME = "ecs-account";
```

Comment:
We should continue to get these values from the [Spring config settings](https://github.com/spinnaker/clouddriver/pull/5070/files#diff-e6a26393446ee1a45534c20fbfbac76438c0240c2e12e3be794b52791674b355L60) (like you were previously) to ensure we're able to read and register account creds/settings from `clouddriver.yml`. 

I think this hardcoded string is leftover from a test which asserted on the account name. We should add back such a check (and use a string instead of a class level var for the 'expected' val) in [`configTest`](https://github.com/spinnaker/clouddriver/pull/5070/files#diff-e6a26393446ee1a45534c20fbfbac76438c0240c2e12e3be794b52791674b355L88) (since the test artifacts in this package rely on that being the correct name) to fast fail in case this value is missing or changed.